server/routes/api/documents/documents.ts

```diff
@@ -1014,15 +1016,7 @@ router.post(
     document.collection = collection;
 
     ctx.body = {
-      data:
-        apiVersion === 2
```

Comment:
Technically a breaking API change but it should only have been used internally in previous versions